Route/DefaultRouteGenerator.php

```diff
@@ -130,16 +130,16 @@ private function getCode(AdminInterface $admin, $name)
             return $admin->getBaseCodeRoute().'.'.$name;
         }
 
+        // someone provide a code, so it is a child
+        if (strpos($name, '.') && array_key_exists($admin->getCode().'|'.$name, $this->caches)) {
```

Comment:
Because must be checked before "someone provide the fullname" tha return the route to the non child admin